examples/image-to-text/run_pipeline.py

```diff
@@ -69,6 +69,7 @@ def main():
         action="store_true",
         help="Whether to perform generation in bf16 precision.",
     )
+    parser.add_argument("--quant_config", action="store_true", help="Enable Quantization to fp8")
```

Comment:
remove

Reply:
It seems this flag https://github.com/HabanaAI/optimum-habana-fork/blob/habana-main/tests/test_text_generation_example.py#L139 confused me.
The expected format is to enable args.quant_config if the QUANT_CONFIG env is set up. Got it

Reply:
Done